<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CatRunUI</class>
 <widget class="QDialog" name="CatRunUI">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>806</width>
    <height>545</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>线束检测</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout" stretch="4,1">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2" stretch="3,1">
     <item>
      <widget class="QLabel" name="imgLabel">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="textLabel">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="text">
        <string>TextLabel</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="yesBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>是</string>
       </property>
       <property name="icon">
        <iconset resource="CatRunUI.qrc">
         <normaloff>:/icons/Resources/apply.png</normaloff>:/icons/Resources/apply.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
       <property name="checkable">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="noBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>否</string>
       </property>
       <property name="icon">
        <iconset resource="CatRunUI.qrc">
         <normaloff>:/icons/Resources/editdelete.png</normaloff>:/icons/Resources/editdelete.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>24</width>
         <height>24</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QPushButton" name="stopBtn">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Minimum" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="text">
        <string>结束检测</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="CatRunUI.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>stopBtn</sender>
   <signal>clicked()</signal>
   <receiver>CatRunUI</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>718</x>
     <y>523</y>
    </hint>
    <hint type="destinationlabel">
     <x>402</x>
     <y>272</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
